server/src/main/java/org/elasticsearch/health/HealthPeriodicLogger.java

```diff
@@ -169,7 +176,7 @@ static HealthPeriodicLogger create(
         BiConsumer<LongGaugeMetric, Long> metricWriter,
         Consumer<ESLogMessage> logWriter
     ) {
-        HealthPeriodicLogger logger = new HealthPeriodicLogger(
```

Comment:
I renamed this, because usually we associate the `logger` variable with the application logger.